python-avd/pyavd/_eos_designs/ip_addressing/__init__.py

```diff
@@ -176,6 +198,20 @@ def p2p_uplinks_ip(self, uplink_switch_index: int) -> str:
 
         return get_ip_from_pool(p2p_ipv4_pool, prefixlen, offset, 1)
 
+    def p2p_uplinks_ipv6(self, uplink_switch_index: int) -> str:
+        """Return Child IP for P2P Uplinks."""
+        uplink_switch_index = int(uplink_switch_index)
+        if template_path := self.shared_utils.ip_addressing_templates.get("p2p_uplinks_ip"):
```

Comment:
We should have a specific v6 template. (should be added in the schema). Same for the others.